packages/editor-ui/src/components/CommunityPackageManageConfirmModal.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+	<Modal
+		width="540px"
+		:name="COMMUNITY_PACKAGE_CONFIRM_MODAL_KEY"
+		:title="getModalContent.title"
+		:eventBus="modalBus"
+		:center="true"
+		:showClose="!isLoading"
+		:beforeClose="onModalClose"
+	>
+		<template slot="content">
+			<n8n-text>{{ getModalContent.message }}</n8n-text>
+			<div :class="$style.descriptionContainer" v-if="this.getCurrentModalAction === COMMUNITY_PACKAGE_MANAGE_ACTIONS.UPDATE">
+				<div :class="$style.descriptionIcon">
+					<n8n-icon icon="info-circle" size="xlarge" />
+				</div>
+				<div :class="$style.descriptionText">
+					<n8n-text>{{ $locale.baseText("We recommend you deactivate workflows that use any of the package's nodes and reactivate them once the update is completed") }}</n8n-text>
```

Comment:
seem like this is not localized correctly

Reply:
🤦‍♂️ Yep, will update.